en/how-to-publish/iati-organisation-identifiers.rst

```diff
@@ -0,0 +1,41 @@
+IATI Organisation Identifiers
+-----------------------------
+In IATI, publishers are identified by their IATI org ID. You can form your IATI org ID by following the steps below. These steps should ensure that your ID unique and will be approved by the IATI tech team.
+
+**1) Identify your registration agency**
+
+A list of registration agencies can be found at: http://org-id.guide/ Check whether your organisation is registered with one of the agencies on this list. If yes, please use the code provided as the first part of your IATI organisation identifier.
+
+If you cannot find the agency your organisation is registered with then please send the name and website of the agency to: contact@org-id.guide. The org-ID team will then start the process of setting up a code for your registration agency.
+
+Example:
+
+  Companies House is a registration agency in the UK. It’s code is: 
+ 
+  GB-COH
+
+  http://org-id.guide/list/GB-COH
+ 
+**2) Your registration number**
+
+The second part of your IATI org ID is the registration number your organisation has been given by the registration agency you have identified above. This can normally be found on your organisation’s registration certificate.
+
+This should be added to your registration agency’s code with a dash (-). The registration number should be written exactly as is appears on your organisation’s registration certificate
+
+Example:
+
+  Companies House has given Development Initiatives the registration number: 06368740   
+  
+  https://beta.companieshouse.gov.uk/company/06368740
+  
+  This is added to the Companies House code to form the IATI org ID:
+ 
+  GB-COH-06368740
```

Comment:
We could work through a similar set of three examples as above.

Reply:
What was the outcome of this?

Reply:
This is fixed with the addition of a second example.